libs/stats/api/src/lib/stats.service.ts

```diff
@@ -0,0 +1,32 @@
+import {Injectable} from '@nestjs/common'
+import {PrismaClient} from '@prisma/client'
+
+@Injectable()
+export class StatsService {
+  constructor(private prisma: PrismaClient) {}
+
+  async getStats() {
```

Comment:
There's a few issues here:

1. All data is loaded aftereachother
2. All articles are counted, even non published ones